jupyter_server/extension/application.py

```diff
@@ -313,13 +313,22 @@ def launch_instance(cls, argv=None, **kwargs):
         # arguments trigger actions from the extension not the server.
         _preparse_command_line(cls)
         # Handle arguments.
-        if argv is not None:
+        if argv is None:
             args = sys.argv[1:]  # slice out extension config.
         else:
             args = []
-        
         # Get a jupyter server instance.
-        serverapp = cls.initialize_server(argv=args)
+        serverapp = cls.initialize_server(
+            argv=args, 
+            load_other_extensions=cls.load_other_extensions
+        )
+        # Log if extension is blocking other extensions from loading.
+        if cls.load_other_extensions:
```

Comment:
@kevin-bates Here's the other logging message.

Reply:
Thanks Zach.  I'm wondering if the phrase "is running without loading other extensions." could get construed as a problematic symptom.  That is, admins may not necessarily know that a given "primary" extension is configured for exclusive use and might wonder - why is it not loading other extensions?  Just thinking that a phrase like the following might be more clear:

"{ext_name} is configured for exclusive use - no other extensions will be loaded."

This way, the intention is clear and not as ambiguous.

Reply:
@Zsailer - I just noticed that the polarity on the condition is not correct.  This should only occur if `load_other_extensions` is False.

Also, I'm thinking it might be better to move this log statement into [`ServerApp.initialize()` as an `else` condition](https://github.com/jupyter/jupyter_server/blob/89c184cdd02cc5a0265be790feeea7599a168a44/jupyter_server/serverapp.py#L1476-L1477) to where the other extensions are being loaded - or is that at too low a level?

Reply:
Oh my. Good catch and sorry for the sloppiness! 🤦‍♂

Reply:
> Also, I'm thinking it might be better to move this log statement into ServerApp.initialize() as an else condition to where the other extensions are being loaded - or is that at too low a level?

My hesitation here is that we'd need to add another argument, `extension_name`, to `ServerApp.initialize()`.  Right now, the ServerApp is completely unaware of ExtensionApps. Adding this argument would break this pattern. Do we think that's ok?

Reply:
Ah - that makes sense - good point.  Let's keep that pattern in tact as long as possible.

Reply:
> "{ext_name} is configured for exclusive use - no other extensions will be loaded."

I think the word "configured" might be confusing since it's not configurable by the user.

How about:
' "jupyter {ext_name}" runs without loading other extensions. If you would like to run {ext_name} with other extensions, try runnig "jupyter server" with {ext_name} enabled.'